README.adoc

```diff
@@ -24,6 +24,8 @@ This web application provides the front door into the Inetgreatly initiative. It
 
 == Local Development
 
+IMPORTANT: The webapp will automatically open (http://localhost:3006) in your browser, however you must hard refresh the page before any content is rendered
```

Comment:
if you want to use however, it should be used as:

...in your browser; however, you must

also missing a period at end

Reply:
@mfrances17 thanks, and doh!